python/tests/unit/connectors/ai/google/google_ai/services/test_google_ai_chat_completion.py

```diff
@@ -275,20 +277,29 @@ async def test_google_ai_streaming_chat_completion_with_function_choice_behavior
 
     google_ai_chat_completion = GoogleAIChatCompletion()
 
+    kernel.add_function(plugin_name="TestPlugin", function=decorated_native_function)
+
+    all_messages = []
     async for messages in google_ai_chat_completion.get_streaming_chat_message_contents(
         chat_history,
         settings,
         kernel=kernel,
     ):
-        assert len(messages) == 1
-        assert messages[0].role == "assistant"
-        assert messages[0].content == ""
-        # Google doesn't return STOP as the finish reason for tool calls
-        assert messages[0].finish_reason == FinishReason.STOP
+        all_messages.extend(messages)
+
+    assert len(all_messages) == 2, f"Expected 2 messages, got {len(all_messages)}"
```

Comment:
Why will there be 2 messages? The mock only has one.

Reply:
This test was broken before. During kernel function invocation it errored out while trying to create the json args because they contained single quotes and not double. The error was never caught or handled because it’s a message added to the chat history. 

This test has two messages now: yielding a function call content message and now with the addition in the PR, a function result content message.